package.json

```diff
@@ -21,10 +21,10 @@
   },
   "scripts": {
     "build": "npm run build:readme && npm run build:js",
-    "prebuild:js": "mkdir -p dist",
-    "build:js": "browserify lib/http.js --outfile dist/chai-http.js --standalone chaiHttp",
+    "prebuild:js": "rm -rf dist",
+    "build:js": "simplifyify lib/http.js --outfile dist/chai-http.js --bundle --minify --debug --standalone chaiHttp",
     "build:readme": "rm -rf README.md && node ./support/readme",
-    "start": "npm run build && http-server -c-1",
+    "start": "npm run build -- --watch & http-server -o -c-1",
```

Comment:
This will background the build process which could be undesirable. It might be better to use something like [`npm-run-all`](https://github.com/mysticatea/npm-run-all/)